lib/src/data/section.dart

```diff
@@ -0,0 +1,24 @@
+import "utils.dart";
+import "chat.dart";
+
+extension type SectionId(String value) {}
+
+class Section {
+  final String section_id;
+  final String CRN;
+  Chat chat;
```

Comment:
```suggestion
  final SectionID id;
  final ChatID chat;
```

Note that the section ID will be the CRN